morty/analytical/ddevolution.py

```diff
@@ -79,14 +79,19 @@ def dist_from_dd_coupling_constant(ddcc, gamma1, gamma2):
 
     Parameters
     ----------
-    r : float
-        Distance between the nuclei in m.
+    ddcc : float
+        Coupling constant in rad/s.
```

Comment:
this seems to have been copy and pysted from the above function. Changed params and returns to accurately describe the function